docs/release_notes/upgrading_to_destinations_v2.md

```diff
@@ -175,12 +175,6 @@ In addition to the changes which apply for all destinations described above, the
 1. [Object and array properties](https://docs.airbyte.com/understanding-airbyte/supported-data-types/#the-types) are properly stored as JSON columns. Previously, we had used TEXT, which made querying sub-properties more difficult.
    - In certain cases, numbers within sub-properties with long decimal values will need to be converted to float representations due to a _quirk_ of Bigquery. Learn more [here](https://github.com/airbytehq/airbyte/issues/29594).
 
-### Snowflake
-
-1. `destination-snowflake` is now case-sensitive, and was not previously. This means that if you have a source stream "users", `destination-snowflake` would have previously created a "USERS" table in your data warehouse. We now correctly create a "users" table.
```

Comment:
@Hesperide fyi - I'm deleting the case-sensitivity thing from snowflake's upgrade guide.